content/en/docs/tasks/access-kubernetes-api/configure-aggregation-layer.md

```diff
@@ -22,6 +22,100 @@ Configuring the [aggregation layer](/docs/concepts/extend-kubernetes/api-extensi
+
+When started with these options, the apiserver will:
+
+1. Use them to authenticate to the extension server.
+2. Create a configmap in the `kube-system` namespace called `extension-apiserver-authentication`, in which it will place the CA certificate and the allowed CNs. These in turn can be retrieved by extension servers to validate requests.
+
+Note that the same client certificate is used by the apiserver to authenticate against _all_ extension servers. It does not create a client certificate per extension server, but rather a single one to authenticate as the apiserver. This same one is reused for all extension server requests. 
+
+#### Original Request Username and Group
+
+When the apiserver proxies the request to the extension server, it informs the extension server of the username and group with which the original request successfully authenticated. It provides these in http headers of its proxied request. You must inform the apiserver of the names of the headers to be used.
+
+* the header in which to store the username via `--requestheader-username-headers`
+* the header in which to store the group via `--requestheader-group-headers`
+* the prefix to append to all extra headers via `--requestheader-extra-headers-prefix`
+
+### Extension Server Authenticate the Request
+
+The extension server, upon receiving a proxied request from the apiserver, must validate that the request actually did come from the apiserver. It does so via:
+
+1. Retrieve the client CA certificate from the configmap in `kube-system`, as described above.
+2. Retrieve the list of allowed names (CNs) from the configmap in `kube-system`, as described above.
+3. Check that the request was signed using a private key whose client certificate was signed using the CA certificate.
+4. Check that the request was signed using a private key whose client certificate has a CN in the list of allowed CNs, unless the list is blank, in which case all CNs are allowed.
+
+If the above passes, then the request is a valid proxied request from the apiserver.
+
+Note that it is the responsibility of the extension server implementation to provide the above. Many do it by default, leveraging the `k8s.io/apiserver/` package. Others may provide options to override it using command-line options.
+
+### Extension Server Authorize the Request
+
+The extension server retrieves the username and group from the headers configured via `--requestheader-username-headers` and `--requestheader-group-headers`. It now needs to validate that this user/group are authorized to execute the given request. It does so by sending a standard [SubjectAccessReview](/docs/reference/access-authn-authz/authorization/) request to the Kubernetes apiserver. 
```

Comment:
extraction of user info from headers is part of the authentication step:
1. verify auth proxy via mTLS 
2. pull user info out of headers

only the subjectaccessreview submission is part of the authorization step

Reply:
Fixed.